articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -67,11 +71,13 @@ stack_cloud = get_cloud_from_metadata_endpoint("https://contoso-azurestack-arm-e
 # https:// is optional in the URL but required on the endpoint.
 credential = DefaultAzureCredential(authority=stack_cloud.endpoints.active_directory)
```

Comment:
This doesn't work for ADFS. 

For example, for ADFS endpoint `https://adfs.redmond.azurestack.corp.microsoft.com/adfs`, `authority='https://adfs.redmond.azurestack.corp.microsoft.com/', tenant_id='adfs'` should be provided as arguments to `DefaultAzureCredential`.

Perhaps Azure Identity should be smart enough to take `authority='https://adfs.redmond.azurestack.corp.microsoft.com/adfs'`? @chlowell